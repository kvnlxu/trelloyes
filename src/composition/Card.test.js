import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';

describe('Card', () => {

  it('renders without crashing', () => {
    // first create a DOM element to render the component into
    const div = document.createElement('div');

    //render the component, this is the actual test, if something is wrong it will fail here
    ReactDOM.render(<Card title="foo" content="bar"/>, div);

    //clean up code
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Card title="foo" content="bar"/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });

});