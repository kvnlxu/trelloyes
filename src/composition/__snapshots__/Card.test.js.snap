// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders the UI as expected 1`] = `
<div
  className="Card"
>
  <h3>
    foo
  </h3>
  <p>
    bar
  </p>
</div>
`;
